src/Barcoder.Renderer.Image/ImageRenderer.cs

```diff
@@ -50,72 +50,89 @@ public void Render(IBarcode barcode, Stream outputStream)
             barcode = barcode ?? throw new ArgumentNullException(nameof(barcode));
             outputStream = outputStream ?? throw new ArgumentNullException(nameof(outputStream));
             if (barcode.Bounds.Y == 1)
-                Render1D(barcode, outputStream);
+            {
+                using (var image = Render1D(barcode))
+                {
+                    image.Save(outputStream, _imageEncoder.Value);
+                }
+            }
+            else if (barcode.Bounds.Y > 1)
+            {
+                using (var image = Render2D(barcode))
+                {
+                    image.Save(outputStream, _imageEncoder.Value);
+                }
+            }
+            else
+                throw new NotSupportedException($"Y value of {barcode.Bounds.Y} is invalid");
+        }
+
+        public Image<L8> Render(IBarcode barcode)
```

Comment:
Types from ImageSharp should be isolated. Why was this public method added? And the code refactored? Doesn't seem required for the beta14 to beta15 upgrade.